packages/ui-kit/src/components/card-elements.tsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import * as designSystem from '../design-system';
+import { css } from '@emotion/react';
+
+/**
+ * the docs ui kit provides the basic elements to compose card UIs but does not provide
+ * the finished card, which would imply how Links are handled and would imply certain context
+ *
+ * A full composition of a card could look like:
+ * <CardContainer {...props}>
+ *  <Link>
+ *    <StackContainer>
+ *        <ImageContainer>{props.image}</ImageContainer>
+ *          <StackContainer>
+ *            <IconContainer>{props.icon}</IconContainer>}
+ *            <StackContainer scale="s">
+ *              <Title smallTitle={props.smallTitle}>{props.title}</Title>
+ *                <BodyContainer>
+ *                  <BodyContent clickable={props.clickable}>
+ *                    {props.children}
+ *                  </BodyContent>
+ *                </BodyContainer>
+ *                <ReadMoreContainer>
+ *                  <ReadMore>
+ *                    <Link href={props.href} noUnderline>
+ *                        {props.textLink}
+ *                    </Link>
+ *                  </ReadMore>
+ *                </ReadMoreContainer>
+ *            </StackContainer>
+ *        </StackContainer>
+ *      </StackContainer>
+ *    </Link>
+ *  </CardContainer>
+ */
+
+// Explanation about the following sizes:
+// https://github.com/commercetools/commercetools-docs-kit/pull/427#discussion_r425442556
+// the "inContentColumn" variation prevents only one card being in one row in regular content (except mobile)
+const cardNarrowMinWidth = '242px';
+const cardRegularMinWidth = '328px';
+const cardRegularMinWidthInContentColumn = '288px';
```

Comment:
Since we are already here: why don't we move these values into the designSystem?